src/utils.js

```diff
@@ -1,173 +1,4 @@
 const fsExtra = require('fs-extra')
-const markdownTable = require('markdown-table')
```

Comment:
Why did you remove all of this stuff? Could still be useful to have our own reports some place, no?

Reply:
Don't think we need to generate `markdown-table`. It is less useful now that validations are done by ACC.